Top-Level Requirements.md

```diff
@@ -25,14 +25,16 @@ Top-Level System Requirements
   * Upon receiving a “Mode-Switch” command, switch to Manual Mode
 * While in Manual Mode, the Robot **shall**:
   * Monitor network traffic for controller information
+  * At all times, keep track of its position
+  * Monitor network traffic for a “Mode-Switch” command
```

Comment:
Since this is listed in both, why not put it in the "While in either mode" section?

Reply:
I agree with jeremy about moving the wording of keep track of position, but I would phrase it as maintain localization, that is the language we have used for the rest of requirements.

I think keeping the language as "Monitor network traffic for controller information" is more prudent. What if we wanted to transmit more kinds of commands than "Mode Switch"? I am not saying get rid of it. I am saying let's keep the more general wording as top level, and make several specific sub requirements.

Reply:
I agree with all 3 points of those, lemme change that

Reply:
I actually didn't even see the either mode part before, so that's useful.